src/System.Diagnostics.Process/tests/ProcessTests.netcoreapp.cs

```diff
@@ -123,5 +129,264 @@ public void BothArgumentSetAndArgumentListSet()
             testProcess.StartInfo = psi;
             Assert.Throws<InvalidOperationException>(() => testProcess.Start());
         }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)]
+        public void Kill_EntireProcessTree_ThrowsPlatformNotSupportedException()
+        {
+            var process = new Process();
+            Assert.Throws<PlatformNotSupportedException>(() => process.Kill(entireProcessTree: true));
+            Assert.Throws<PlatformNotSupportedException>(() => process.Kill(entireProcessTree: false));
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
+        public void Kill_EntireProcessTree_True_ProcessNotStarted_ThrowsInvalidOperationException()
+        {
+            var process = new Process();
+            Assert.Throws<InvalidOperationException>(() => process.Kill(entireProcessTree: true));
```

Comment:
out of curiosity - not expecting any fixes - what's the issue on Uap?

Reply:
I didn't implement on UAP because I thought UAP doesn't give access to parent process IDs. (I don't see an API to access the parent PID at https://docs.microsoft.com/en-us/uwp/win32-and-com/win32-apis.)

Does that sound correct or am I missing something? :-)

Reply:
Sounds good, I wouldn't worry about UAP yet, if we need to fix it we can fix it separately 😄

Reply:
:-)